pkgs/development/compilers/julia/generic.nix

```diff
@@ -60,6 +60,10 @@ stdenv.mkDerivation rec {
     substituteInPlace deps/curl.mk \
       --replace-fail 'cd $(dir $<) && $(TAR) jxf $(notdir $<)' \
                      'cd $(dir $<) && $(TAR) jxf $(notdir $<) && sed -i "s|/usr/bin/env perl|${lib.getExe buildPackages.perl}|" curl-$(CURL_VER)/scripts/cd2nroff'
+  ''
+  + lib.optionalString (lib.versionOlder version "1.12") ''
```

Comment:
Is there a reason not to include version 1.12 here as well? #450240 is working on 1.12 support.